index.html

```diff
@@ -1,52 +1,314 @@
+							<p class="work_item_header">Tonic</p>
+							<div class="work_history">
+								<p class="client">
+									<span class="client_name">Canopy</span>
+								</p>
+								<p class="counter">
+									<img src="images/Counter.png" alt="counter icon">                            
+								</p>
+								<p class="role">
+									<span class="role_name">Back End Dev</span>
+								</p>
+								<p class="counter">
+									<img src="images/Counter.png" alt="counter icon">                            
+								</p>
+								<p class="year">
+									<span class="date">2015</span>
+								</p>
+							</div>   
+						</div>       		
+						<p class="work_summary">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+						<div class="tags">
+							<p class="html">
+								<span>html</span>
+							</p>
+							<p class="css">
+								<span>css</span>
+							</p>
+							<p class="js">
+								<span>javascript</span>
+							</p>
+						</div>
+						<div class="project_link">
+							<p>
+								<a href="#">See Project</a>
+							</p>
+						</div>
+					</div> 
+				</div>
+
+				<div class="work_item">
+					<img src="images/Snapshoot Portfolio (3).png" alt="snapshot portifolio image">
+					<div class="work_details">
+						<div class="project">
+							<p class="work_item_header">Multi-Post Stories</p>
+							<div class="work_history">
+								<p class="client">
+									<span class="client_name">Canopy</span>
+								</p>
+								<p class="counter">
+									<img src="images/Counter.png" alt="counter icon">                            
+								</p>
+								<p class="role">
+									<span class="role_name">Back End Dev</span>
+								</p>
+								<p class="counter">
+									<img src="images/Counter.png" alt="counter icon">                            
+								</p>
+								<p class="year">
+									<span class="date">2015</span>
+								</p>
+							</div>   
+						</div>       		
+						<p class="work_summary">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+						<div class="tags">
+							<p class="html">
+								<span>html</span>
+							</p>
+							<p class="css">
+								<span>css</span>
+							</p>
+							<p class="js">
+								<span>javascript</span>
+							</p>
+						</div>
+						<div class="project_link">
+							<p>
+								<a href="#">See Project</a>
+							</p>
+						</div>
+					</div> 
+				</div> 
+				</div>
+			</section>
+			<section class="about_myself">
+				<div class="about_me_container">
+					<div class="about_me">
+						<h1>About<br>Myself</h1>
+						<p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+						<div class="connect">
+							<div class="connect_text2">
+								<p>LET'S CONNECT</p>    
```

Comment:
:red_circle: LET'S CONNECT is a title, It should be in a header tag hx. Please make sure you use a header tag hx here